superset/commands/utils.py

```diff
@@ -102,3 +107,79 @@ def get_datasource_by_id(datasource_id: int, datasource_type: str) -> BaseDataso
         )
     except DatasourceNotFound as ex:
         raise DatasourceNotFoundValidationError() from ex
+
+
+def validate_tags(
+    object_type: str,
```

Comment:
couldn't this be ENUM of resource `CHART || DASHBOARD || QUERY`

Reply:
def could! I was already using `ObjectType` for the `update_tags` method, but here thought about going with `string` because I'm using `security_manager.can_access` to check if the user has the required perm, which is case sensitive. We have two options here:
1. Create a new `enum` with:
``` python
class AssetTypes(enum.Enum):
    CHART = "Chart"
    DASHBOARD = "Dashboard"
```
2. Use the existing `ObjectType` and then call `can_access` using `capitalize()`:
``` python
if not security_manager.can_access("can_tag", object_type.name.capitalize()):
```

@hughhhh would you have any preferences?

Reply:
Let's go with #2 using `ObjectType` enum